attestation_demo/EnclaveInitiator/ReadMe.txt

```diff
@@ -7,14 +7,14 @@ Enclave Project Wizard has created this Enclave1 project for you as a starting p
 This file contains a summary of what you will find in each of the files that make up your project.
```

Comment:
Do we need this file in the repo at all?

Reply:
No, I don't think so.

Reply:
Removed in [bf683a5](https://github.com/NationalSecurityAgency/pelz/pull/107/commits/bf683a517abca63bb90cbdb4f5cd88478dbfc48f)